src/common/networks.ts

```diff
@@ -65,13 +65,13 @@ export const supportedNetwork: {
     currency: "MATIC",
     gasStation: gasStation("https://gasstation.polygon.technology/v2"),
   },
-  [NetworkCmdName.Maticmum]: {
-    explorer: "https://mumbai.polygonscan.com",
-    provider: defaultInfuraProvider("maticmum"),
-    networkId: 80001,
-    networkName: NetworkCmdName.Maticmum,
+  [NetworkCmdName.Amoy]: {
+    explorer: "https://www.oklink.com/amoy",
+    provider: jsonRpcProvider("https://polygon-amoy.infura.io/v3/bb46da3f80e040e8ab73c0a9ff365d18"),
```

Comment:
Cause we are still on ethers v5 so we connect manually like the Sepolia example above (line 55)